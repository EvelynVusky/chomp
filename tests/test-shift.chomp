int main() {
    bit a = {{1}};
    nibble b = {{0101}};
    byte c = {{11100010}};
    word d = {{0111000110011110}};
    
    println(a << 2);
    println(b << 24);
    println(c >> 3);
    println(d >> 11);
    println(b >> 24);
    
    return 0;
}