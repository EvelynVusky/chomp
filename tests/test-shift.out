0
0000
00011100
0000000000001110
0000
